src/rust/bitbox02-rust/src/hww/api/ethereum/sign.rs

```diff
@@ -266,7 +278,11 @@ async fn verify_standard_transaction(
         .await?;
     }
 
-    let address = super::address::from_pubkey_hash(&recipient);
+    let address_verify = super::address::from_pubkey_hash(&recipient);
+    let address = format!("0x{}", request.address());
```

Comment:
why does the request.address() not contain `0x`? that prefix is part of the address.

Reply:
Okay, I will add `0x` when the request is first created